environment.yml

```diff
@@ -17,8 +17,8 @@ channels:
 - fastai
 dependencies:
 - python==3.6.8
-- pytorch>=1.2.0
-- torchvision>=0.3.0
+- pytorch==1.2.0
+- torchvision==0.4.0
```

Comment:
Can we undo this change, i.e. not edit environment.yml (since that might break our devops tests) @angusrtaylor